### ISSUE TITLE
Similar to spectral, add "hsl"

### ISSUE THREAD

**Issue by maneetgoyal** [manufac-analytics/reactjs-calendar-heatmap#17]
```js
import {hsl} from 'd3';
const get_color = (val) => hsl(360 * val, 0.85, 0.7).toString();
```

"Val" ranges from 0 to 1

**Comment by qur786** [comment]
> import {hsl} from 'd3'; const get_color = (val, offset = 0.0) => hsl(360 * val, 0.85, 0.7).toString();
> 
> "Val" ranges from 0 to 1

saturation and lightness should be the constants mentioned in the comment (0.85, 0.7) @maneetgoyal sir?

**Comment by qur786** [comment]
And the use case of `offset` sir ?

**Comment by maneetgoyal** [comment]
> > import {hsl} from 'd3'; const get_color = (val, offset = 0.0) => hsl(360 * val, 0.85, 0.7).toString();
> > "Val" ranges from 0 to 1
> 
> saturation and lightness should be the constants mentioned in the comment (0.85, 0.7) @maneetgoyal sir?

Yes.

**Comment by maneetgoyal** [comment]
> And the use case of `offset` sir ?

Can be removed. I'll update the function.

**Comment by maneetgoyal** [comment]
closed by #18
